source/_assets/less/main.less

```diff
@@ -1,31 +1,75 @@
-@import "base";
-@import "colors";
-@import "typography";
-@import "z-index";
-@import "grid";
-@import "flex";
-@import "images";
-
-// Components
-@import "forms";
-@import "panel";
-@import "list-group";
-@import "buttons";
-@import "navbar";
-@import "svg-icon";
-@import "nav-list";
-@import "documentation-page";
-@import "code-editor";
-@import "docsearch";
-@import "files";
-@import "logo";
-
-// Utilities
-@import "backgrounds";
-@import "layout";
-@import "spacing";
-@import "borders";
-@import "utilities";
-@import "responsive-utilities";
-@import "transitions";
-@import "animations";
+@tailwind preflight;
+@tailwind components;
+@tailwind utilities;
+
+@import 'typography';
+@import 'files';
```

Comment:
What is `files`? I don't see it anywhere.

Reply:
This has been updated in #33 , see [here](https://github.com/tightenco/jigsaw-site/blob/8004d5e7004e0bb782e388eccf783226ada96d19/source/_assets/less/files.less).